RELEASE_NOTES.md

```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## stable-1.20-4.4.0, v4.4.0
+
+### kube-oidc-proxy
+  - * bump kube-oidc-proxy to 0.3.0
```

Comment:
```suggestion
  - Bumps kube-oidc-proxy to 0.3.0 to resolve "kubectl log" latency issues (COPS-7123)
```

Reply:
Please update the PR release notes and I'll run the generator again.

Reply:
✅